index.html

```diff
@@ -0,0 +1,119 @@
+                </p>
+            </section>
+            <hr>
+            <section id="gastronomy">
+                <h2>Gastronomy</h2>
+                <a href="">Back to the top</a>
+                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTreItholsHoekLuXnbDDxHcimSZXhU74HrYg&usqp=CAU" alt="" height="190px">
+                <p>
+                    The regional Minas Gerais' food and the now internationally known drink of cachaça are very popular and highly rated in the capital. Belo Horizonte is internationally known as the "capital of neighborhood bars."
+                </p>
+            </section>
+            <hr>
+            <section id="more">
+                <h2 >More</h2>
+                <p><a href="https://en.wikipedia.org/wiki/Category:Football_clubs_in_Belo_Horizonte">Soccer </a> ⚽️</p>
+                <p><a href="https://www.artchitectours.com/tour/belo-horizonte-modernism/">Architecture </a> 🌇</p>
+                <p><a href="https://www.brazilnaturetours.com/brazil-tours/belo-horizonte">Nature </a> 🌲</p>
+            </section>
+            <hr>
+            <section id="contact">
+                <h2>Contact</h2>
+                <p>📞 <strong>+55 31 99345611</strong> </p>
+                <p>📧 <strong>tourism@belohorizonte.com</strong> </p>
+            </section>
+        </main>
+        <hr>
+        <footer>
+            <p><em>Made by Eduardo Melo</em></p>
+            <a href=""><p>Back to the top</p></a>
+        </footer>
+        <!-- <header>
+            <h1>Welcome to my page</h1>
+            <nav>
+                <ul>
+                    <li>Home</li>
+                    <a href="#about">
+                        <li>About</li>
+                    </a>
+                    <li>Contact</li>
+                    <li>Carrer</li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <section>
+                <h2 id="about">About Me</h2>
+                <img src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/grapefruit-slice-332-332.jpg" alt="eduardo´s picture">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Reprehenderit, adipisci molestiae. Doloribus dolorum animi odio accusamus consectetur nam praesentium vel explicabo, modi libero similique tempora numquam id quas rerum possimus!
+                </p>
+            </section>
+            <section>
+                <h2>Things I like to do</h2>
+                <ul>
+                    <li><strong>Music</strong></li>
+                    <li>Travel</li>
+                    <li>Food</li>
+                </ul>
+            </section>
+            <section>
+                <h2>Social Media Links</h2>
+                <a href="">Facebook</a>
+                <a href="">LinkedIn</a>
+                <a href="">Twitter</a>
+            </section>
+        </main>
+        <footer><em>made by Eduardo</em> ⚽️</footer> -->
```

Comment:
comments!. You can delete this because this will be also in your served html file.